apps/src/code-studio/header.js

```diff
@@ -168,10 +170,44 @@ function setupReduxSubscribers(store) {
 setupReduxSubscribers(getStore());
 
 function setUpGlobalData(store) {
-  store.dispatch(asyncLoadUserData());
+  fetch('/api/v1/users/current', {
```

Comment:
Should we document this somewhere? I can't tell how widespread this application of fetch might be but if we're moving toward fetch over ajax and the default credentials are still different for older browsers I imagine we'll run into this again?

Reply:
I definitely agree. One way to handle this would be to not use fetch directly but write our own wrapper around it so that these kind of configurations will be set for it every time it's used. I'll start a conversation about this in apps-devs tomorrow!